ir/src/transition_constraints/degree.rs

```diff
@@ -0,0 +1,87 @@
+// The [TransitionConstraintDegree] struct and documentation contained in this file were duplicated
+// from the Winterfell STARK prover library (https://github.com/novifinancial/winterfell), which is
+// licensed under the MIT license. The implementation in this file is a subset of the Winterfell
+// code.
+
+// The original code is available in the Winterfell library in the `air` crate:
+// https://github.com/novifinancial/winterfell/blob/main/air/src/air/transition/degree.rs
```

Comment:
If this is intended to be a module level comment, i think the syntax is `//!`.

Reply:
Fixed, thank you!